sections/featured-collection.liquid

```diff
@@ -109,6 +109,7 @@
               {%- assign placeholder_image = 'product-apparel-' | append: forloop.rindex -%}
               {% render 'card-product',
                 show_vendor: section.settings.show_vendor,
+                image_shape: section.settings.image_shape,
                 placeholder_image: placeholder_image
```

Comment:
```suggestion
                image_shape: section.settings.image_shape,
                media_aspect_ratio: section.settings.image_ratio,
                placeholder_image: placeholder_image
```
If you choose to go with the second option I mentioned then add the `media_aspect_ratio: section.settings.image_ratio,` here and remove `height: auto;` from `component-card.css` file on line 303.

Reply:
Yes I like this option! It looks way better and reflects an actual image! Thank you!